src/sql/workbench/common/sqlWorkbenchUtils.ts

```diff
@@ -19,3 +19,38 @@ export function getSqlConfigValue<T>(workspaceConfigService: IConfigurationServi
 	let config = workspaceConfigService.getValue<{ [key: string]: any }>(ConnectionConstants.sqlConfigSectionName);
 	return config ? config[configName] : undefined;
 }
+
+/**
+ * Wraps provided string using Unicode character U+000A LINE FEED (LF) that qualifies as line break to wrap text in title attributes (tooltips).
+ * @param str string to be wrapped
+ * @param maxWidth max width to be allowed for wrapped text
+ * @returns wrapped string
+ */
+export function getWrappedString(str: string, maxWidth: number): string {
+	let newLineStr = `
+`;
+	let res = '';
+	while (str.length > maxWidth) {
```

Comment:
are you concerned if the input is ever undefined?  or that is a precondition?

Reply:
It's expected to contain text, but I can add a check for safety!